systemProperties.adoc

```diff
@@ -0,0 +1,25 @@
+== System Properties
+Below, a number of relevant System Properties are listed.
+
+
+NOTE: All boolean properties below are read using link:https://javadoc.io/static/net.openhft/chronicle-core/2.23ea13/net/openhft/chronicle/core/Jvm.html#getBoolean-java.lang.String-[net.openhft.chronicle.core.Jvm.getBoolean(java.lang.String)], and so are enabled if either `-Dflag` or `-Dflag=true` or `-Dflag=yes`
+
+.System properties
+[cols=4*, options="header"]
+|===
+| Property Key | Default | Description | Java Variable Name (Type)
+| chronicle.network.callOnMissedHeartbeatOnDisconnect |`false` | If enabled, calls HeartBeatListener when we do not receive a heartbeat | _CALL_MISSED_HEARTBEAT_ON_DISCONNECT_ (boolean)
+| client.timeout | 500 | Number of milliseconds after which a client is timed out if they cannot to a host port | _pausePeriodMs_ (int)
+| connectionStrategy.pauseMillisBeforeReconnect | 500 | Allows control of a backoff strategy, and returns how long in milliseconds to pause before attempting reconnect | pauseMillisBeforeReconnect (long)
+| connectionStrategy.pause.max.secs | 5 | Controls how frequently the ConnectionStrategy.connect method is called | _maxPauseSec_ (long)
+| connectionStrategy.pause.min.secs | 5 | Controls how frequently the ConnectionStrategy.connect method is called | _minPauseSec_ (long)
+| connectionStrategy.socketConnectionTimeoutMs | 1 | Minimum time before message displayed: "Timed out attempting to connect to " + socketAddress | socketConnectionTimeoutMs (int)
```

Comment:
So this is the timeout for one single attempt at connection (how long we wait for a socket to open). `client.timeout` seems to be the overall timeout (how long we keep retrying for).

i.e. if the strategy is to retry every X seconds, each individual attempt will timeout after `connectionStrategy.socketConnectionTimeoutMs` and we will stop retrying after `client.timeout`. It's probably worth making that clear the distinction between the two timeouts?